.github/workflows/codecov.yml

```diff
@@ -0,0 +1,31 @@
+name: Codecov Workflow
+on:
+  pull_request:
+    branches:
+      - '**'
+  push:
+    branches:
+      - master
+jobs:
+  build:
+    name: Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@master
+      - name: Use Node.js 18.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 18.x
```

Comment:
We still don't have node 18 in the master :grin:

Reply:
After merging Gatsby 5 PR we will have it

Reply:
<img width="827" alt="image" src="https://github.com/binary-com/deriv-com/assets/129206554/230ddded-23bb-45d7-8b8c-3d120eb1bd0f">
This is from `staging.yml` also couple of more `yaml` files I saw version `18` so I thought It would be better make keep it `18`

Reply:
Alrighty :smile: